### DISCUSSION TITLE
[Craft 5 alpha] Blocktype too close to tabs and too similar to inactive tab title in inline matrix blocks

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#14188] [General]
In the latest Craft 5 alpha (alpha 8) we have tabs working in inline Matrix blocks (great 🥳).

One minor UI thing that I noticed is that the Block Type (or Entry Type) looks just like an inactive tab title and, since the tabs are positioned right next to it, that might cause some confusion.

![image](https://github.com/craftcms/cms/assets/19857599/6247d7ed-1be2-4b5d-a945-8057f72c27d2)

Maybe a simple adjustment like _having a bigger gap between blocktype and tabs_ could be enough to aleviate this?

**Comment by brandonkelly** [comment]
Just widened the gap a bit for the next release (21e0550e2d91d0d0e50eb18ad0febab6250d5380). I’ve also moved the checkbox to the other side of the titlebar and pulled in the entry type icon, if there is one, which will help reinforce that the first label is the type name, not a tab (#14187).

<img src="https://github.com/craftcms/cms/assets/47792/d41136e9-3ba9-45d2-b62b-6a0cc0253a83" width="291" alt="The top-left of a Matrix field block, showing the new style changes.">

**Reply by thupsi** [reply]
Perfect! 👌
